### PR TITLE
Made page more responsive

### PR DESCRIPTION
**Pull request by TurnbulA**
Have upped the font size to make more readable.
Each form entry has its own line.
Still need to make footer fall in place

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,58 +216,70 @@ <h1>Lorem ipsum dolor sit amet consectetur adipisicing elit<span
         <div class="footer">
             <h1 class="footer-text">iPaidThat</h1>
             <div class="footer-content">
-                <div class="row">
+                <div class="row ">
```
Comment (mali891) [inline index.html]: Is this space intentional?

File: style.css [hunk 2]
```diff
@@ -374,19 +380,67 @@ option{
...
+
+	.footer-content{
+		margin: 0;
+	}
+	.footer-row{
+		width: 100%;
+		margin: 0;
+	}
+	.list-1{
+		margin-left: -40rem;
```
Comment (mali891) [inline style.css]: This doesn't feel right. What is it for?

File: index.html (hunk 1 above)
Comment (TurnbulA) [reply index.html]: No, must've been a mis-click on the space

File: style.css [hunk 3]
```diff
@@ -374,19 +380,67 @@ option{
...
+		text-align: center;
+	}
+	.list-2{
+		margin-top: -34rem;
```
Comment (mali891) [inline style.css]: This doesn't feel right. What is it for?

File: style.css [hunk 4]
```diff
@@ -374,19 +380,67 @@ option{
...
+		margin-right: -40rem;
```
Comment (mali891) [inline style.css]: This doesn't feel right. What is it for?

File: style.css (hunk 2 above)
Comment (mali891) [reply style.css]: @TurnbulA ?

File: style.css (hunk 3 above)
Comment (mali891) [reply style.css]: @TurnbulA ?

File: style.css (hunk 4 above)
Comment (mali891) [reply style.css]: @TurnbulA ?

**Pull request closed without merging**